assets/dev/js/editor/utils/jquery-html5-dnd.js

```diff
@@ -205,19 +205,40 @@
 				return;
 			}
 
+			// Make sure that the previous placeholders are removed before inserting a new one.
+
+			// console.log( 'placeholders' );
+			// console.log( elementsCache.$element );
+			// console.log( 'parent placeholders' );
+			// // console.log( elementsCache.$element[0].parent() );
+			// console.log( elementsCache.$element.parent() );
+			// elementsCache.$element.find( '.elementor-widget-placeholder' ).remove();
+
 			// Fix placeholder placement for Container with `flex-direction: row`.
 			const $currentElement = $( currentElement ),
-				isRowContainer = $currentElement.parents( '.e-container--row' ).length,
```

Comment:
Comment by Evyatar: maybe make it more specific:

currentelement closest e-container check if has class e-container--row